jmx-metrics/docs/target-systems/wildfly.md

```diff
@@ -0,0 +1,76 @@
+# WildFly Metrics
+
+The JMX Metric Gatherer provides built in WildFly metric gathering capabilities.
```

Comment:
Is there a ~jmx monitoring reference you can link to like is done in the other docs?

Reply:
Unlike the other JMX integrations, this one has been harder to link to official documentation. The WildFly docs outline how to connect through JMX (https://docs.jboss.org/author/display/WFLY/JMX%20subsystem%20configuration.html), but I've been unable to find a resource listing the available mbeans. The majority of my research was done through manually searching the hierarchy in jconsole and finding the most common metrics.

There are 3rd party resources out there that outline some of the more important mbeans (such as Datadog for example), but I felt it wasn't appropriate to link to another vendor's/product's documentation.

Reply:
What you linked may be better than nothing and is helpful in suggesting this is a supported/provided interface? Unfortunate no better guides are in the rh docs.

Reply:
Agreed. It is unfortunate. I went ahead and added that link, with verbiage similar to what Solr did.